addons/website_sale/views/templates.xml

```diff
@@ -1451,6 +1449,9 @@
                             </div>
                         </td>
                         <td class="text-center td-price" name="price">
+                            <t t-set="combination" t-value="line.product_id.product_template_attribute_value_ids + line.product_no_variant_attribute_value_ids"/>
+                            <t t-set="combination_info" t-value="line.product_id.product_tmpl_id._get_combination_info(combination, pricelist=website_sale_order.pricelist_id, add_qty=line.product_uom_qty)"/>
+
```

Comment:
Is this change intended? Why?

Reply:
If we want to display the base unit price when summarizing the cart using `combination_info['base_unit_price']`, it must exist and be updated via the `_get_combination_info` function. These lines existed in earlier versions and , I think, were removed because if you don't use `combination_info`, they are not necessary.